install_cli.php

```diff
@@ -647,6 +664,49 @@ class FakeLogger { public function __call($m, $a) { } }
 		DataTool::$team_sets_array = $team_sets;
 	}
 
+	if ($module == 'ACLRoles' && !empty($_SESSION['tba'])) {
+		$tbaAccess = 72; //This value related to ACL_ALLOW_SELECTED_TEAMS(Owner & Selected Teams) constant
+		$roleActions = array(
+			'Accounts' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Contacts' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Leads' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Quotes' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Opportunities' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Bugs' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Cases' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'KBContents' => array('delete' => $tbaAccess, 'export' => $tbaAccess)
+		);
+
+		$ACLRolesData = array();
+		$result = $GLOBALS['db']->query("SELECT id FROM acl_roles WHERE id LIKE 'seed-ACLRoles%'");
+		while($row = $GLOBALS['db']->fetchByAssoc($result)){
+			$ACLRolesData[$row['id']] = $row['id'];
+			$ACLRole = BeanFactory::retrieveBean('ACLRoles', $row['id']);

```

Comment:
This retrieve is not necessary and it's very heavy. Take a look at the [setAction](https://github.com/sugarcrm/Mango/blob/master/sugarcrm/modules/ACLRoles/ACLRole.php#L65-L88). It doesn't use `$this`. Thus you could just create an empty bean outside of the `while` and call `setAction` on it:
```php
$roleBean = BeanFactory::getBean('ACLRoles');
while (/* ... */) {
    //...
    $roleBean->setAction($ACLRole->id, $actionId['id'], $access_override);
    //...
}
```